pkgs/sdk_triage_bot/lib/src/prompts.dart

```diff
@@ -36,6 +37,7 @@ area-web: Use area-web for Dart web related issues, including the DDC and dart2j
 Don't make up a new area.
 Don't use more than one area- label.
 If it's not clear which area the issue should go in, don't apply an area- label.
+Take your time when considering which area to triage the issue into.
```

Comment:
I would put even more pressure on the LLM ;)

Reply:
I do wonder if this phrase will make it spend more consideration on the response. I have heard that phrases like 'show your work' will make it produce more tokens, which effectively means that it's using more computation, which may then yield a better result.

I'm not sure that it'll help us here since the classification response is only ever going to yield ~1/2 dozen tokens (`area-foo, type-bar`).